notification/twilio/voice.go

```diff
@@ -187,7 +187,7 @@ func (v *Voice) Send(ctx context.Context, msg notification.Message) (*notificati
 		Params:         make(url.Values),
 	}
 
-	prefix := fmt.Sprintf("This is %s", cfg.ApplicationName())
+	prefix := fmt.Sprintf("Hello! This is %s. <break strength=\"x-strong\" time=\"700ms\"/>", cfg.ApplicationName())
```

Comment:
We will need to update all the messages, too, with this change.

Currently, the prefix is spoken as part of the introduction, "This is {application} with alert notifications {details}" we'll want to address the text in the switch statement below.

Maybe drop a message in the chatroom about the proposal to ensure buy-in. My understanding is you're proposing we restructure voice messages from "{Appname & message type} {details} {options}" to "{appname} {details} {options}" or maybe the intent is "{appname} {type} {details} {options}"?

In any case, it's worth a few minutes to chat about the how and why so we can review and ensure all calls are consistent and all reviewers know what to look for.

Reply:
Actually, this is just a minor wording an vocal pacing update.  The structure for phone calls will still remain `{Appname} {details} {options}`, and that is untouched.  The current Greeting is `This is GoAlert {details} {options}` which is all run together and sounds like it's all one long sentence.  This change simple adjusts the greeting to `Hello! This is GoAlert. <700ms pause> {details} {options}`, which adds a natural vocal separation between the greeting, and the message details.

Reply:
If you look at line 196, for example, we are appending the reason for the call to the prefix, which places it after the pause as a broken sentence.

The current greeting is:
`This is GoAlert with an alert notification. {details} {options}`

With this change it is:
`Hello! This is GoAlert. <700ms pause> with an alert notification. {details} {options}`

I think it should go before the pause, or be rephrased to be a complete sentence.

For example:
`Hello! This is GoAlert with an alert notification. <700ms pause> {details} {options}`

Reply:
Updated all of the pauses to do `Hello! This is GoAlert with an alert notification. <700ms pause> {details} {options}`